internal/aws-sdk-go/go.mod

```diff
@@ -1,10 +1,5 @@
 module github.com/aws/aws-sdk-go
 
-require (
```

Comment:
Why are we changing the internal package's go.mod? Was it not working without the change?

Reply:
This was auto-updated.
After removing internal/aws-sdk-go, this will be deleted. Will update when the other PR is merged.